NEWS.md

```diff
@@ -9,6 +9,8 @@ shiny 1.6.0.9000
 
 ### New features and improvements
 
+* All uses of `list(...)` have been replaced with `rlang::list2(...)`. This means that you can use trailing `,` without error and use rlang's `!!!` operator to "splice" a list of argument values into `...`. We think this'll be particularly useful for passing a list of `tabPanel()` to their consumers (i.e., `tabsetPanel()`, `navbarPage()`, etc). For example, `tabs <- list(tabPanel("A", "a"), tabPanel("B", "b")); navbarPage(!!!tabs)`. (#3315 and #3328)
```

Comment:
```suggestion
* All uses of `list(...)` have been replaced with `rlang::list2(...)`. This means that you can use trailing `,` without error and use rlang's `!!!` operator to "splice" a list of argument values into `...`. (#3315 and #3328)
```

Since it doesn't provide any new benefit, I think it's better to not over advertise it.

Reply:
`!!!` inside `tabsetPanel()` is new (was merged earlier today), but `!!!` inside `tagList()` isn't.

That said, I'd be OK with removing it, but I'll leave that up to @wch